app/scripts/modules/kubernetes/src/v2/pipelines/stages/runJob/KubernetesV2RunJobStageConfig.tsx

```diff
@@ -14,6 +27,11 @@ export class KubernetesV2RunJobStageConfig extends React.Component<IStageConfigP
     credentials: [],
   };
 
+  public findSelectedArtifact(id: string): IExpectedArtifact {
```

Comment:
where is this used?

Reply:
A remnant of a previous approach. Will remove.